DESCRIPTION

```diff
@@ -73,7 +73,7 @@ URL: https://pharmaverse.github.io/admiral/, https://github.com/pharmaverse/admi
 Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.1
+RoxygenNote: 7.2.3
```

Comment:
Should this be changing in a PR?

Reply:
No, this was not intended. I have reverted it.